index.html

```diff
@@ -12,9 +11,7 @@
     <div class="book-container">
       <img src="images/book-cover.png" class="book-cover" alt="book-cover">
       <div class="book-detail">
-        <h1 class="title">
-          Título del libro <span class="placeholder genre">(género)</span>
-        </h1>
+        <h1 class="title">Título del libro <span class="placeholder genre">(género)</span></h1>
```

Comment:
Avoid nesting tags inside `h1` tags. For better SEO, try to keep them clean. You can use a flex display and move the `span` next to the `h1`